### ISSUE TITLE
Tabbed blocks as normal markdown

### ISSUE THREAD

**Issue by alisondy** [google/docsy#550]
Tabbed blocks forces me to use a code block, Would be nice if there was an option for me to use a mix of normal markdown text and code blocks like in the following screenshot

![image](https://user-images.githubusercontent.com/19580905/117681794-5d220780-b1aa-11eb-9e9c-e34953785b95.png)

**Comment by alisondy** [comment]
whoops pressed enter too quickly

**Comment by LisaFC** [comment]
@deining can you take a look? Thanks!

**Comment by alisondy** [comment]
Sorry I should have included this earlier aswell! this is the code associated for making a tabbed view like pictured!
https://github.com/tektoncd/website/tree/main/layouts/shortcodes

**Comment by deining** [comment]
Hi Alison,

glad you find my tabbed pane shortcodes useful!

> Tabbed blocks forces me to use a code block,
> Would be nice if there was an option for me to use
> a mix of normal markdown text and code blocks

When I started coding the shortcodes, the first version treated content as markdown (as requested from you). Later on, I changed this and decided to treat the inner content as code. Offering the opportunity to treat content either as code or as markdown makes certainly sense. One idea is to add another shortcode `tab-code` and let the `tab` shortcode treat content as markdown (or html). This might break existing use cases of the current shortcode, however, Another idea is to specify the content type as named parameter. Let me reflect on this a bit. I'll come back to your proposal, but due to other commitments, this might take a while.
